libsolidity/ExpressionCompiler.cpp

```diff
@@ -596,18 +596,18 @@ void ExpressionCompiler::appendCompareOperatorCode(Token::Value _operator, Type
 
 		switch (_operator)
 		{
-		case Token::GTE:
+		case Token::GreaterThanOrEquals:
```

Comment:
I would drop the `s` in the `Equals`. In `GreaterThanOrEquals` it sounds especially weird. Furthermore, `Token::Equals` denotes the symbol and not the relation.